src/spacegroup.f90

```diff
@@ -0,0 +1,27 @@
+! stand alone driver to read in lattice vectors and atomic basis
```

Comment:
This file is incomplete, i.e., it doesn't actually do anything and should probably be removed.

Reply:
I'm going to leave it there until the "inactive" stuff is done. I can't remember how I was using it. Now that the "compare" stuff is done, I can try and finish up that.